solr/test-framework/src/java/org/apache/solr/util/RestTestBase.java

```diff
@@ -152,6 +152,12 @@ public static void assertQ(String request, String... tests) {
     }
   }
 
+  public static void assertHead(String request, int expectedStatusCode) throws IOException {
+    String response = restTestHarness.head(request);
```

Comment:
String is passed around to avoid tying `RestTestBase` to the actually httpclient implementation.